quickwit/quickwit-lambda/src/indexer/model.rs

```diff
@@ -38,6 +38,7 @@ impl IndexerEvent {
             IndexerEvent::S3(event) => [
                 "s3://",
                 event.records[0].s3.bucket.name.as_ref().unwrap(),
+                "/",
```

Comment:
it feels like the kind of things `test_s3_event_uri` should have caught. What gives?

Reply:
I added a make command to run the Lambda tests, but these tests are not running in CI (for performance reasons)

Reply:
okay. then we should make sure to run them manually when they may be relevant